analyzers/tests/SonarAnalyzer.UnitTest/Extensions/AssignmentExpressionSyntaxExtensionsTests.cs

```diff
@@ -27,6 +27,356 @@ namespace SonarAnalyzer.UnitTest.Extensions
     [TestClass]
     public class AssignmentExpressionSyntaxExtensionsTests
     {
+        [TestMethod]
+        public void AssignmentExpressionSyntaxExtensions_MapAssignmentArguments_TupleElementsAreExtracted()
```

Comment:
`AssignmentExpressionSyntaxExtensions_` is already in the class name, repeats in every test and makes it hard to identify what method/case is tested => please remove it everywhere.